dogen/plugins/cct.py

```diff
@@ -47,6 +35,15 @@ def setup_cct(self, version):
 
         return cctdist
 
+    def get_cct_plugin(self, cfg):
+        if 'dogen' not in cfg:
+            cfg['dogen'] = {}
+        if 'plugins' not in cfg['dogen']:
+            cfg['dogen']['plugins'] = {}
+        if 'cct' not in cfg['dogen']['plugins']:
+            cfg['dogen']['plugins']['cct'] = {}
```

Comment:
In general, this method should be removed and above  one line should be moved to `prepare` function.